### PR TITLE
Script for Defect 2445

### PR DESCRIPTION
**Pull request by TMyza**
ATF Test Scripts to check [#2445](https://github.com/smartdevicelink/sdl_core/issues/2445)

Summary
A set of test scripts to verify that issue is fixed

This PR is **not ready** for review.
This is **draft** version of the script.

### CODE REVIEW

File: test_scripts/Defects/5_0/2445_SDL_doesn_t_resume_status_of_subscribtion_on_wayPoint_related_data.lua [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    common.getMobileSession():ExpectResponse(cid, { success = true , resultCode = "SUCCESS" })
+    common.getMobileSession():ExpectNotification("OnHashChange")
+end
+
+local function onWayPointChange()
+    common.getHMIConnection():SendNotification("Navigation.OnWayPointChange", notifParams)
+    common.getMobileSession():ExpectNotification("OnWayPointChange", notifParams)
+end
+
+local function closeMobileSession()
```
Comment (aderiabin) [inline test_scripts/Defects/5_0/2445_SDL_doesn_t_resume_status_of_subscribtion_on_wayPoint_related_data.lua]: @TMyza According test description mobile session should be closed with unexpected disconnect of mobile device. Current function unregisters mobile application correctly without disconnect.

File: test_scripts/Defects/5_0/2445_SDL_doesn_t_resume_status_of_subscribtion_on_wayPoint_related_data.lua [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+runner.Step("PTU", common.policyTableUpdate, { pTUpdateFunc })
+runner.Step("Activate App", common.activateApp, { 1 })
+runner.Step("SubscribeWayPoints", subscribeWayPoints)
+runner.Step("On Way Point Change", onWayPointChange)
+runner.Step("Close mobile session", closeMobileSession)
+runner.Step("Clean sessions", cleanSessions)
+
+-- [[ Test ]]
+runner.Title("Test")
+runner.Step("Register App", common.registerAppWOPTU, { 1 })
```
Comment (aderiabin) [inline test_scripts/Defects/5_0/2445_SDL_doesn_t_resume_status_of_subscribtion_on_wayPoint_related_data.lua]: @TMyza Test scenario does not check resumption of HMI level of unexpected disconnected mobile application and does not check restoring of its subscription to onWayPointChange HMI event.
Script is incorrect.

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.
Review and appropriate updates going be done till 1st of November.
Thank you.

**Comment by GetmanetsIrina** [comment]
@aderiabin, all comments are addressed, please review

**Comment by dboltovskyi** [comment]
Closed due to not actual of original issue.

**Pull request closed without merging**